ceph.spec.in

```diff
@@ -33,6 +30,9 @@ Source0:	http://ceph.com/download/%{name}-%{version}.tar.bz2
 %if 0%{?fedora} || 0%{?rhel}
 Patch0:		init-ceph.in-fedora.patch
 %endif
+#################################################################################
+# distro-unconditional dependencies
```

Comment:
"dependencies that apply across all distro families" sounds fine to me (not too verbose)